cartridges/int_klaviyo_sfra/cartridge/controllers/Klaviyo.js

```diff
@@ -2,88 +2,83 @@
 
 var server = require('server');
 
-// Use the following for CSRF protection: add middleware in routes and hidden field on form
+/* API Includes */
+var StringUtils = require('dw/util/StringUtils');
 
-/* eslint-disable */
-var csrfProtection = require("*/cartridge/scripts/middleware/csrf");
-/* eslint-enable */
+/* Script Modules */
+var klaviyoUtils = require('*/cartridge/scripts/klaviyo/utils');
+var viewedProductData = require('*/cartridge/scripts/klaviyo/eventData/viewedProduct');
+var viewedCategoryData = require('*/cartridge/scripts/klaviyo/eventData/viewedCategory');
+var searchedSiteData = require('*/cartridge/scripts/klaviyo/eventData/searchedSite');
+var KLCheckoutHelpers = require('*/cartridge/scripts/klaviyo/checkoutHelpers');
 
-var Logger = require('dw/system/Logger');
 
-/* API Includes */
+/***
+ *
+ * NOTE: The Klaviyo-Event route exists to support event tracking on pages whose OOTB SFCC controllers are cached by default.
+ * To avoid caching event data, the Klaviyo-Event route is called via remote include in KlaviyoTag.isml.
```

Comment:
I believe the file name in this comment needs to be updated - KlaviyoTag.isml was deleted

Reply:
good catch!  can correct.